src/transferwise_clj/core.clj

```diff
@@ -1,87 +1,49 @@
 (ns transferwise-clj.core
-  (:require [clj-http.client :as client]
-            [transferwise-clj.constants :as const]))
-
-;BASIC FUNCTIONS AND PREDICATES
-(defn is-greater [l-rate r-rate] (> l-rate r-rate))
-
-(defn diff-between [l-value r-value] (- l-value r-value))
-
-(defn is-best-rate 
-  "Check if the rate is best than the last fetched one and the best over all time"
-  [rate last-rate best-rate] 
-  (and (is-greater rate last-rate) 
+  (:require [transferwise-clj.data.core  :refer [new-telegram-map]]
+            [transferwise-clj.data.atoms :refer [best-rate new-best-rate]]
+            [transferwise-clj.constants  :refer [last-transaction quote]]
+            [transferwise-clj.api.core   :refer [transferwise-get-quote telegram-send-advise]]))
+
+;PREDICATES
+(defn is-greater 
```

Comment:
Since Clojure is dynamic, we like to use `?` at the end of functions names that return a bool. In this case: `is-greater?`